### PR TITLE
Added support for RGBW NeoPixels

### PR DESCRIPTION
**Pull request by eboskma**
This adds support for RGBW NeoPixels with the SK6812 controller.

### CODE REVIEW

File: examples/ws2812_i2s/ws2812_i2s_colour_loop.c [hunk 1]
```diff
@@ -69,15 +71,23 @@ static void demo(void *pvParameters)
...
 {
     uart_set_baud(0, 115200);
+    struct sdk_station_config config = {
+        .ssid = "Loading...",
+        .password = "morays59924_howitzer",
+    };
+
+    /* required to call wifi_set_opmode before station_set_config */
+    sdk_wifi_set_opmode(STATION_MODE);
+    sdk_wifi_station_set_config(&config);
```
Comment (UncleRus) [inline examples/ws2812_i2s/ws2812_i2s_colour_loop.c]: I don't think that WiFi-related things are necessary here

**Comment by UncleRus** [comment]
I believe that example should be left untouched (RGB mode by default)

File: examples/ws2812_i2s/ws2812_i2s_colour_loop.c (hunk 1 above)
Comment (eboskma) [reply examples/ws2812_i2s/ws2812_i2s_colour_loop.c]: Oops, that is not meant to be there...

**Comment by eboskma** [comment]
I've updated the example to use RGB pixels.

**Comment by ourairquality** [comment]
I see a few people that seem to have not base patches off the current master branch. This can be corrected with a sequence such as: `git checkout master` `git fetch upstream` `git merge upstream/master` `git push` `git checkout <mybanch>` `git rebase master` `git push -f <mybranch>`. If your situation is more complex and you need to move just a few patches from some working branch then look at `git rebase --onto master ...`

**Comment by UncleRus** [comment]
Squashed & merged

**Pull request merged**